File: KIOSKK/McD/item.cpp
#include "item.h"

Item::Item(QString name, int price)
    :name(name), price(price){

}
Item::Item(QString kind, QString name, QString option, int price)
    :kind(kind), name(name), option(option), price(price){

}
Item::Item(QString name, QString option, int price)
    :name(name), option(option), price(price){

}
QString Item::getName() const{
    return name;
}
int Item::getPrice() const{
    return price;
}

File: Order_APP(Android_java_client)/app/src/main/java/com/example/myapplication/Model/Item.java
package com.example.myapplication.Model;

import java.io.Serializable;

public class Item implements Serializable {
    String kind;
    String name;
    String option;
    int price;

    public String getKind() {
        return kind;
    }

    public void setKind(String kind) {
        this.kind = kind;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getOption() {
        return option;
    }

    public void setOption(String option) {
        this.option = option;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }
}
